test/test_helper.rb

```diff
@@ -8,6 +8,8 @@
 require 'database_cleaner'
 require 'date'
 require 'simplecov'
+require 'mocha/test_unit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.